src/Microsoft.SqlTools.ServiceLayer/QueryExecution/QueryExecutionService.cs

```diff
@@ -789,7 +789,7 @@ internal async Task HandleCopyResultsRequest(CopyResultsRequestParams requestPar
                             }
                         }
                         // Add line break if this is not the last row in all selections.
-                        if (rowIndex + pageStartRowIndex != lastRowIndex && (!builder.ToString().EndsWith(Environment.NewLine) || !requestParams.AvoidNewLineAfterTailingLineBreak))
+                        if (rowIndex + pageStartRowIndex != lastRowIndex && (!builder.ToString().EndsWith(Environment.NewLine) || !requestParams.SkipNewLineAfterTrailingLineBreak))
```

Comment:
Why is this even a request param anyways? It's a global setting so you should probably just be getting it from the general config IMO

Reply:
See https://github.com/microsoft/sqltoolsservice/blob/a37093a773ab92d646768005be1b834a217c66a8/src/Microsoft.SqlTools.ServiceLayer/SqlContext/SqlToolsSettings.cs for examples of these settings. STS will get notified whenever a config value is updated so that reduces the load on our end to send this stuff ourselves.

Reply:
fixed